Explorer/Assets/DCL/Multiplayer/Profiles/Entities/RemoteEntities.cs

```diff
@@ -125,7 +129,9 @@ private Entity CreateCharacter(RemoteProfile profile, World world)
         {
             var transform = transformPool.Get()!;
             transform.name = $"REMOTE_ENTITY_{profile.WalletId}";
-            transform.transform.SetParent(null);
```

Comment:
Why were we previously setting the parent as null?

Reply:
we were taking them from the pool and "unparenting" them from it, so they would appear on the hierarchy... that was not a good approach tbh because it cluttered the hierarchy...